engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/expression/errors/Redefined.scala

```diff
@@ -230,7 +230,7 @@ object Redefined {
 
     /** @inheritdoc */
     override def showCode(indent: Int): String =
-      s"(Redefined (Conversion ${targetType.map(_.showCode() + ".").getOrElse("")}from $sourceType))"
+      s"(Redefined (Conversion ${targetType.map(_.showCode() + ".").getOrElse("")}from ${sourceType.showCode()}))"
```

Comment:
Small irrelevant fix - before printing the redefined conversion as code was leaking a lot of information by using `toString` instead of `showCode` for the `sourceType`. With that it is more compact and closer to what I think `showCode` is supposed to be.